docs-src/concepts/what-is-a-search-attribute.md

```diff
@@ -29,23 +29,28 @@ A Temporal Cluster has a set of default Search Attributes already available.
+| WorkflowType               | Keyword  | The type of Workflow.                                                                                                                                                        |
+| WorkflowId                 | Keyword  | Identifies the Workflow Execution.                                                                                                                                           |
+| ExecutionStatus            | Keyword  | The current state of the Workflow Execution.                                                                                                                                 |
+| StartTime                  | Datetime | The time at which the Workflow Execution started.                                                                                                                            |
+| CloseTime                  | Datetime | The time at which the Workflow Execution completed.                                                                                                                          |
+| ExecutionTime              | Datetime | Same as StartTime for the most cases but different for cron Workflows and retried Workflows. For them it is the time at which the Workflow Execution actually begin running. |
+| RunId                      | Keyword  | Identifies the current Workflow Execution Run.                                                                                                                               |
+| ExecutionDuration          | Int      | The time needed to run the Workflow Execution. Available only for closed Workflows.                                                                                          |
+| HistoryLength              | Int      | The number of events in the history of Workflow Execution. Available only for closed Workflows.                                                                              |
+| StateTransitionCount       | Int      | The number of times that Workflow Execution has persisted its state. Available only for closed Workflows.                                                                    |
+| TaskQueue                  | Keyword  | Task Queue used by Workflow Execution.                                                                                                                                       |
+| TemporalChangeVersion      | Keyword  | If Workflow versioning is enabled, list of change/version pairs will be stored here.                                                                                         |
+| BinaryChecksums            | Keyword  | List of binary Ids of Workers that run the Workflow Execution.                                                                                                               |
```

Comment:
You might as well add BuildIds (to come with the new versioning feature) here while you're editing things.
@bergundy could suggest a good wording. Something like
"BuildIds | Keyword | List of Worker Build Ids that have processed the Workflow Execution, or Build Id of a desired compatible version. Will obsolete BinaryChecksums."